amplify/ts-code/src/db/Schemas.ts

```diff
@@ -62,6 +62,24 @@ export interface HistorySchema {
     timestamp: number
 }
 
+export const SCHEDULE_TABLE: string = process.env.STORAGE_SCHEDULE_NAME
+/**
+ * @param id Random Time Related Unique Key, where the first part of the key is the creation time.
+ * @param reserver Email of person reserving Item.
+ * @param itemId ID of Item.
+ * @param startTime Time reservation starts.
+ * @param endTime Time reservation end.
+ * @param notes Optional notes about this action.
+ */
+export interface ScheduleSchema {
+    id: string,
+    reserver: string,
```

Comment:
Can we make it consistent and make it `borrower`?

Reply:
Done!